proxy/src/rate_limiter/limiter.rs

```diff
@@ -293,7 +297,7 @@ mod tests {
             .map(|s| s.parse().unwrap())
             .collect();
         RateBucketInfo::validate(&mut rates).unwrap();
-        let limiter = EndpointRateLimiter::new(rates);
+        let limiter = BucketRateLimiter::new(rates);
```

Comment:
Any reason not to use WakeupComputeRateLimiter here?

Reply:
Nope. Just missed it during the refactor